src/current/cockroachcloud/tools-page.md

```diff
@@ -62,11 +62,16 @@ To monitor the health of metrics export, you can [create a custom Monitor](#moni
 
 ### View and configure dashboards
 
-Open your Datadog [Dashboard List](https://docs.datadoghq.com/dashboards/#dashboard-list) and click `CockroachDB Dedicated Overview`. This sample dashboard presents a high-level view of SQL performance and latency, and information about resource consumption to help aid in capacity planning. It provides the ability to drill down to specific nodes (identified by a `(node, region)` tag pair) within your cluster.
+Open your Datadog [Dashboard List](https://docs.datadoghq.com/dashboards/#dashboard-list). There are two out of the box dashboards that present CockroachDB metrics
```

Comment:
suggest changing "out of the box" to "sample dashboards" like below

suggest adding a colon before the bulleted list "CockroachDB metrics: "

Reply:
updated